compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -825,14 +825,39 @@ renderValue world name = \case
         renderField (Field label mbValue) =
             renderValue world (name ++ [TL.toStrict label]) (fromJust mbValue)
 
+templateConName :: Identifier -> Maybe (LF.Qualified LF.TypeConName)
+templateConName (Identifier _ (TL.toStrict -> qualName)) = do
+  (modName, tpl) <- case T.splitOn ":" qualName of 
+    [mdN, defN] -> Just (LF.ModuleName (T.splitOn "." mdN), LF.TypeConName [defN])
+    _ -> error "Bad definition"
+  return (LF.Qualified LF.PRSelf  modName tpl)
```

Comment:
Instead of `PRSelf` we should use the package name in the first argument of `Identifier` above (the `_`).